lib/nodes/decorators/RepeatWhen.ts

```diff
@@ -1,33 +1,13 @@
 'use strict';
 
-import {ResultCodes} from '../../utils/ResultCodes';
 import {Base} from '../Base';
-import {Decorator} from './Decorator';
+import {RepeatWhenNTimes, ResultConditional} from "./RepeatWhenNTimes";
```

Comment:
should be single quotes here

Reply:
My webstorm defaults to "" instead of ' '.  I need to figure out how to change that setting.